### ISSUE TITLE
10x slowdown when compiling with -g

### ISSUE THREAD

**Issue by s-ludwig** [ldc-developers/ldc#4354]
At least on Windows, I'm experiencing an extreme slowdown for 1.32.0, as opposed to previous versions, when compiling with `-g`. For a particular library, complation took around 33 s before and 340 s with 1.32.0. Running with `-ftime-trace` shows 222 s spent in "optimizing", as opposed to 650 ms without `-g`.

**Comment by kinke** [comment]
Ouch, sounds really bad. Might be an oversight for the 'new' LLVM pass manager setup used for LLVM 15+.

**Comment by kinke** [comment]
Yep, seems definitely related to the new pass manager. Compile time is around 84 secs with LDC v1.31.0 (LLVM 14) with `-g`; the build with `-g -passmanager=new` is taking ages.

**Comment by kinke** [comment]
Further tests have shown that it's the combination of new pass manager, `-g` and NO `-disable-verify`. Fix incoming.
